src/megaapi.cpp

```diff
@@ -2213,7 +2213,11 @@ void MegaApi::getAccountDetails(MegaRequestListener *listener)
 
 void MegaApi::getSpecificAccountDetails(bool storage, bool transfer, bool pro, MegaRequestListener *listener)
 {
-    pImpl->getAccountDetails(storage, transfer, pro, false, false, false, listener);
+    assert(storage || transfer || pro);
+    if (storage || transfer || pro)
```

Comment:
Every single method accepting a `listener` is expected to create a request and eventually receive the callbacks `onRequestStart()/Finish()`. This is a no no no, since the app will NOT receive any feedback and might enter in a forever waiting state.

Reply:
Done!